near-sdk/src/collections/tree_map.rs

```diff
@@ -153,6 +208,21 @@ where
     }
 
     /// Iterate entries in ascending order: given key (exclusive) to max (inclusive)
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let mut map: TreeMap<u32, u32> = TreeMap::new(b"m".to_vec());
+    /// let one: Vec<u32> = vec![10, 20, 30, 40, 50,45, 35, 25, 15, 5];
+    /// for x in &one {
+    ///     map.insert(x, &42);
+    /// }
+    /// assert_eq!(
+    ///     map.iter_from(29).collect::<Vec<(u32, u32)>>(),
```

Comment:
optional: perhaps change this to iterating from 30 to show off the exclusive nature of the start index?